src/github.com/getlantern/flashlight/ui/resources.go

```diff
@@ -1,7 +0,0 @@
-// This file will be overwritten by make genassets.
```

Comment:
The side effect of this is not being able to compile flashlight like this `source setenv.bash && go build github.com/getlantern/flashlight/main` without running genassets before. Perhaps we can find another way to check if assets have been generated instead of relying on the existence of this file? The `src/github.com/getlantern/lantern-ui/dist` directory should not exists if assets have not been generated, for instance.